.github/workflows/migration-check.yml

```diff
@@ -0,0 +1,45 @@
+name: Check if all migrations have been performed
+on:
```

Comment:
this should run on pull request and branch pushes, rather than tag pushes, so we can catch errors before we publish a release